src/zenml/cli/stack_components.py

```diff
@@ -1362,154 +1365,21 @@ def flavor_group() -> None:
-            cli_utils.declare(
-                f"Annotation stats for '{dataset_name}' dataset:", bold=True
-            )
-            cli_utils.declare(f"Total annotation tasks: {total_task_count}")
-            cli_utils.declare(f"Labeled annotation tasks: {labeled_task_count}")
-            cli_utils.declare(
-                f"Unlabeled annotation tasks: {unlabeled_task_count}"
-            )
-
-        @dataset.command("delete")
-        @click.argument("dataset_name", type=click.STRING)
-        @click.option(
-            "--all",
-            "-a",
-            "all",
-            is_flag=True,
-            help="Use this flag to delete all datasets.",
-            type=click.BOOL,
-        )
-        @click.pass_obj
-        def dataset_delete(
-            annotator: "BaseAnnotator", dataset_name: str, all: bool
-        ) -> None:
-            """Delete a dataset.
-
-            If the --all flag is used, all datasets will be deleted.
-
-            Args:
-                annotator: The annotator stack component.
-                dataset_name: Name of the dataset to delete.
-                all: Whether to delete all datasets.
-            """
-            cli_utils.declare(f"Deleting your dataset '{dataset_name}'")
-            dataset_names = (
-                annotator.get_dataset_names() if all else [dataset_name]
-            )
-            for dataset_name in dataset_names:
-                annotator.delete_dataset(dataset_name=dataset_name)
-                cli_utils.declare(
-                    f"Dataset '{dataset_name}' has now been deleted."
-                )
-
-        @dataset.command(
-            "annotate", context_settings={"ignore_unknown_options": True}
-        )
-        @click.argument("dataset_name", type=click.STRING)
-        @click.pass_obj
-        def dataset_annotate(
-            annotator: "BaseAnnotator", dataset_name: str
-        ) -> None:
-            """Command to launch the annotation interface for a dataset.
-
-            Args:
-                annotator: The annotator stack component.
-                dataset_name: Name of the dataset
-
-            Raises:
-                ValueError: If the dataset does not exist.
-            """
-            cli_utils.declare(
-                f"Launching the annotation interface for dataset '{dataset_name}'."
-            )
-            try:
-                annotator.get_dataset(dataset_name=dataset_name)
-                annotator.launch(
-                    url=annotator.get_url_for_dataset(dataset_name)
-                )
-            except ValueError as e:
-                raise ValueError("Dataset does not exist.") from e
-
-
 def register_all_stack_component_cli_commands() -> None:
     """Registers CLI commands for all stack components."""
     for component_type in StackComponentType:
         register_single_stack_component_cli_commands(
             component_type, parent_group=cli
         )
+
```

Comment:
Do we need these calls actually in the for loop and check for the `component_type`, or can we simply call all of them after the call to `register_all_stack_component_cli_commands()` below?

Reply:
Good point. Fixed.